### DISCUSSION TITLE
Disable/Enable RAG mid conversation?

### DISCUSSION THREAD

**Discussion by rvkwi** [open-webui/open-webui#3175] [Q&A]
Is there a way to turn off/disable the RAG mid conversation?

Sometimes it would be useful if you could use the RAG only for one or a few responses. Especially if it's something vaguely related in a conversation about a different topic (say you are trying to troubleshoot/solve thing A, run into something relating to a thing B and want to use the RAG to help along with B. But really once the info is in it and B is clear, you might want to turn B off again to get back to A. But from now on any further message will trigger the RAG on the B doc.)

Is there a way to do this somewhere?

**Comment by ArthurJ** [comment]
Related to #3556

**Reply by maurizioaiello** [reply]
It is not relevant that RAG can be disabled by admin, as stated in #7209 , because it would be very useful to disable RAG in a single chat, and not as a system setting for all the models and all the chat.
It is very important to have RAG for data discovery and inference for example, but also to be able to say "Summarize the document" while the system is setp for RAG. This is a very important feature to be implemented chat per chat (not model by model)
